### ISSUE TITLE
Horizontal Clusterize for Table Cells?

### ISSUE THREAD

**Issue by vpetreski** [NeXTs/Clusterize.js#94]
Hi guys,

I am wondering if there is an easy way to solve problem I have with Clusterize or I'd have to create my own solution for this:

I have a table which can have, lets say, 100.000 rows. I use Clusterize to have that working smoothly.

My problem now is that every row can have 50 cells, each one having some UI components inside. Only with 20 rows in the table I end up with 20 * 50 = 1000 cells with few components in each one, which is causing browser to slowdown and freeze.

So, basically, I need exactly the same solution that Clusterize is providing for "vertical" problem, but now applied horizontally - to do the same magic when user scrolls horizontally for cells (TD).

Any thoughts?

Thanks,
Vanja

**Comment by kpowick** [comment]
Can you post an example of screen shot of what this looks like?  I'm trying to imagine a scenario where  I would want so many UI components in a design.

My thoughts would be to simply display data values in the table, but have a pop-up/modal editor with the appropriate UI components to change the values of a selected table row.

**Comment by vpetreski** [comment]
![screenshot](https://cloud.githubusercontent.com/assets/1615782/22677496/ffff93f4-ecc0-11e6-9be1-12260e13af2b.png)

It is requested to be like this by the client, because it's critical to do updates very quickly by moving from cell to cell, using shortcuts.

**Comment by kpowick** [comment]
You could have a fixed row with the UI controls that represents the row currently selected/highlighted by the user.  They would then still be able to use shortcuts to move very quickly to the desired UI control of the selected row.

This would also allow for a feature whereby the user could select several rows and change the contents of the selected rows simultaneously.

**Comment by vpetreski** [comment]
That makes a lot of sense, thanks @kpowick for your suggestions!
